lib/customisation.nix

```
@@ -120,6 +151,64 @@ rec {
       else result);
 
 
+  /* Call the function in the file `fn` with the required
+    arguments automatically.  This should be preferred over callPackage
+    when the result does not need to expose a .override function.  The function
```

```suggestion
    when the result is not a derivation.  The function
```

Not sure which wording is more accurate or succinct